modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPOrganization.macro

```diff
@@ -0,0 +1,147 @@
+definition {
+
+	@summary = "Add a new organization"
+	macro createAnOrganization {
+		EVPNavigator.goToNavItem(navItem = "Organization");
+
+		Button.click(button = "Create an Organization");
+	}
+
+	@summary = "Fill in all required fields on the Organization Form"
+	macro fillOrganizationForm {
+		WaitForPageLoad();
+
+		if (isSet(organizationName)) {
+			Type(
+				key_inputValue = "Organization Name",
+				locator1 = "EVPUtils#INPUT_FIELD",
+				value1 = ${organizationName});
+		}
+
+		if (isSet(organizationSite)) {
+			Type(
+				key_inputValue = "Organization Site/Social Media Link",
+				locator1 = "EVPUtils#INPUT_FIELD",
+				value1 = ${organizationSite});
+		}
+
+		if (isSet(smallDescription)) {
+			Type(
+				key_inputValue = "Small Description",
+				locator1 = "EVPUtils#INPUT_FIELD",
+				value1 = ${smallDescription});
+		}
+
+		if (isSet(street)) {
+			Type(
+				key_inputValue = "Street",
+				locator1 = "EVPUtils#INPUT_FIELD",
+				value1 = ${street});
+		}
+
+		if (isSet(city)) {
+			Type(
+				key_inputValue = "City",
+				locator1 = "EVPUtils#INPUT_FIELD",
+				value1 = ${city});
+		}
+
+		if (isSet(state)) {
+			Type(
+				key_inputValue = "State",
+				locator1 = "EVPUtils#INPUT_FIELD",
+				value1 = ${state});
+		}
+
+		if (isSet(country)) {
+			Type(
+				key_inputValue = "Country",
+				locator1 = "EVPUtils#INPUT_FIELD",
+				value1 = ${country});
+		}
+
+		if (isSet(zip)) {
+			Type(
+				key_inputValue = "Zip",
+				locator1 = "EVPUtils#INPUT_FIELD",
+				value1 = ${zip});
+		}
```

Comment:
Hi @joelrocha68, if you replace the variable `zip` for `zipCode`, will be more clear that it's related to Postal Code. What do you think?